src/betterproto/__init__.py

```diff
@@ -1225,7 +1229,9 @@ def _include_default_value_for_oneof(
             meta.group is not None and self._group_current.get(meta.group) == field_name
         )
 
-    def load(self: T, stream: BinaryIO, size: Optional[int] = None) -> T:
+    def load(
+        self: T, stream: BinaryIO, size: Optional[int] = None, delimited: bool = False
+    ) -> T:
```

Comment:
Possibly remove `size`?

```suggestion
    def load(self: T, stream: BinaryIO, *, delimited: bool = False) -> T:
```

Reply:
The `delimited` parameters have been replaced by use of `betterproto.SIZE_DELIMITED` inside of `size` for loading and `delimit` for dumping.